### PR TITLE
Create form

### PR DESCRIPTION
**Pull request by hannahgolightly**
The front end add item page and css for the form

### CODE REVIEW

File: add_item.php [hunk 1]
```diff
@@ -14,12 +14,61 @@
...
+				<label for="served">Served: </label>
+				<select id="served" name="served" size="1">
+					<option value="on_rocks">Poured over ice</option>
+					<option value="frozen">Frozen</option>
+					<option value="fruity">Over crushed ice</option>
+					<option value="refreshing">Straight up</option>
+				</select>
+			</div>
+			<div class="one_field">
+			<input class="button" type="submit" value="Submit"/>
```
Comment (HiredMussel) [inline add_item.php]: Indent this line for readability

File: style.css [hunk 2]
```diff
@@ -46,7 +48,7 @@ ul {
 
 /* top container with title in */
 .container_top {
-    height: 100px;
+    height: 180px;
```
Comment (HiredMussel) [inline style.css]: You mentioned responsiveness the other day. The issue with the titles overflowing might be coming from the fact that this container has a fixed height. What happens if you don't explicitly define the height here and set overflow to auto?

File: style.css [hunk 3]
```diff
@@ -85,13 +86,49 @@ ul {
...
 }
 
 
 /* add item page, the container holding the form section */
 .form_container {
-    height: 400px;
-    width: 100%;
-    margin-bottom: 50px; 
+    height: 750px;
+    width: 60%;
```
Comment (HiredMussel) [inline style.css:97]: Width 60% is going to get difficult to read on small screens. Consider adding a min-width entry.

**Review by HiredMussel — APPROVED** [review]
A couple of nitpicks, but your frontend looks good from here. Go ahead and merge this!

File: style.css (hunk 2 above)
Comment (hannahgolightly) [reply style.css]: great thanks- this fixed my issue!

File: style.css (hunk 3 above)
Comment (hannahgolightly) [reply style.css:97]: added this in thanks

**Pull request merged**